ingenialink/dictionary.py

```diff
@@ -898,6 +899,152 @@ class DictionaryV2(Dictionary):
 
     dict_interface: Optional[str]
 
+    MON_DIST_STATUS_REGISTER = "MON_DIST_STATUS"
+
+    monitoring_disturbance_registers: Dict[
+        Interface, List[Union[EthercatRegister, EthernetRegister, CanopenRegister]]
+    ] = {
+        Interface.ECAT: [
```

Comment:
Did you think about adding the registers in the communication-specific dictionary class?

Reply:
Done!